package.json

```diff
@@ -45,38 +46,95 @@
   },
   "homepage": "https://github.com/angular/devkit",
   "dependencies": {
+    "@angular-devkit/build-optimizer": "0.0.41",
+    "@angular/common": "^5.2.1",
```

Comment:
We don't want them, no. But to testing the webpack build facade right now is done through unit tests instead of e2e tests, so we don't have a way to create peerDep setups. 

Once we have a way to setup e2e tests these can go away.